client/app/common/navbar/navbar.html

```diff
@@ -11,13 +11,13 @@
             </li>
           </ul>
         </div>
-        <ul class="nav navbar-nav" ng-if="$ctrl.auth.getAuthState()">
+        <!-- <ul class="nav navbar-nav" ng-if="$ctrl.auth.getAuthState() && UsersCrtl.subTitle">
```

Comment:
subTitle is not working. It's in the old/ directory though. Should we add subTitle logic? If so, I can rework and just open up another PR for this aspect.